scripts/package.sh

```diff
@@ -1,26 +1,32 @@
 PKG_ROOT=$1
 OUT_DIR=$2
 
+early_exit() {
```

Comment:
While working on the library I had missed some Rust error, this allows the script to exit early instead of trying to run all 4 builds.